app/scripts/components/common/browse-controls/use-browse-controls.ts

```diff
@@ -8,11 +8,10 @@ export enum Actions {
   SEARCH = 'search',
```

Comment:
Is this hook only going to be used for Stories Hub? (If so, can we put a comment about it?)